### DISCUSSION TITLE
I don't understand why my express website isn't working.

### DISCUSSION THREAD

**Discussion by 1PV1** [vercel/community#1506] [Help]
### Summary

I have a simple express website with a single page that uses html, css and js, everything that is for client-side is in the public folder, but I don't understand why it isn't running the express code, on the website just appears the express code as shown below:
![image](https://user-images.githubusercontent.com/98609931/218322427-3ec643ca-2efb-4d52-adeb-96d6b6754142.png)
Also what port should I use?

### Additional information

```js
require('dotenv').config();
const express = require('express');

const app = express();

app.use(express.static('public'));

app.get('/', (req, res) => {
  res.sendFile(__dirname + '/public/home.html');
});

app.listen(process.env.PORT || 10000, () => {
  console.info(`[${new Date(Date.now()).toLocaleString('ro-RO', {timeZone: 'Europe/Bucharest'})}][\x1b[32minfo\x1b[0m]: Website is up!`)
});
```

**Comment by amyegan** [comment]
Hi @1PV1! Did you follow a guide, such as [Using Express.js with Vercel](https://vercel.com/guides/using-express-with-vercel)? Or do you have a different project structure?
